docs/port-requirements.md

```diff
@@ -0,0 +1,60 @@
+# Port Requirements
+
+To operate properly, KubeSpray requires some ports to be opened. If the network is configured with firewall rules, it is needed to ensure infrastructure components can communicate with each other through specific ports.
+
+Ensure the following ports required by KubeSpray are open on the network and configured to allow access between hosts. Some ports are optional depending on the configuration and usage.
+
+## Kubernetes
+
+### Control plane
+
+| Protocol | Port   | Description     |
+|----------|--------| ------------    |
+| TCP      | 22     | ssh for ansible |
+| TCP      | 2379   | etcd client port|
+| TCP      | 2380   | etcd peer port  |
+| TCP      | 6443   | kubernetes api  |
+| TCP      | 10250  | kubelet api     |
+| TCP      | 10257  | kube-scheduler  |
+| TCP      | 10259  | kube-controller-manager  |
+
+### Worker node(s)
+
+| Protocol | Port       | Description     |
+|----------|--------    | ------------    |
+| TCP      | 22         | ssh for ansible |
+| TCP      | 10250      | kubelet api     |
+| TCP      | 30000-32767| kube nodePort range |
+
+refers to: [Kubernetes Docs](https://kubernetes.io/docs/reference/networking/ports-and-protocols/)
+
+## Calico
+
+If Calico is be used, it requires:
+
+| Protocol | Port       | Description   |
+|----------|--------    | ------------  |
+| TCP      | 179        | Calico networking (BGP) |
+| UDP      | 4789       | Calico CNI with VXLAN enabled |
+| TCP      | 5473       | Calico CNI with Typha enabled  |
+| IPENCAP / IPIP | -    | Calico CNI with IPIP enabled  |
+
+refers to: [Calico Docs](https://docs.tigera.io/calico/latest/getting-started/kubernetes/requirements#network-requirements)
+
+## Cilium
+
+If Cilium  is be used, it requires:
+
+| Protocol | Port     | Description   |
+|----------|--------  | ------------  |
+| UDP      | 8472     | Cilium CNI with VXLAN  |
```

Comment:
There's 4244 for hubble as well if it is enabled.

Reply:
Thanks @MrFreezeex 

The Port has beed add to the docs :-)